sagemaker-remote-function/quick_start/quick_start.ipynb

```diff
@@ -0,0 +1,265 @@
+    "\n",
+    "Before running the samples here, please follow the quick start guide to set up the environment properly, including downloading and installing the private version of SageMaker Python SDK.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install the dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install -r ./requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "## Invoke Python function as a SageMaker job\n",
+    "\n",
+    "There are two ways users could invoke a function as a job.\n",
+    "\n",
+    "* Using the decorator. When the decorated function is invoked, it executes synchronously"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "from sagemaker.remote_function import remote\n",
+    "\n",
+    "@remote\n",
+    "def divide(x, y):\n",
+    "    print(f\"Calculating {x}/{y}\")\n",
+    "    return x / y\n",
+    "\n",
+    "divide(3, 2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "* Using the RemoteExecutor APIs. They follow the pattern of the Python [concurrent.futures](https://docs.python.org/3/library/concurrent.futures.html) APIs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "def divide(x, y):\n",
+    "    print(f\"Calculating {x}/{y}\")\n",
+    "    return x / y\n",
+    "\n",
+    "from sagemaker.remote_function import RemoteExecutor\n",
+    "\n",
+    "with RemoteExecutor(max_parallel_job=1, keep_alive_period_in_seconds=30) as executor:\n",
+    "    futures = [executor.submit(divide, x, 2) for x  in [1, 2, 3]]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "[future.result() for future in futures]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "## Customize the decorator settings\n",
+    "\n",
+    "In the example above, the @remote decorator and RemoteExecutor look for configurations in the following places\n",
+    "\n",
+    "* The global configuration file at `<user’s home folder>.sagemaker/defaults/sdk-default-config.yaml`\n",
+    "* The local configuration file at the current directory where the decorated function is invoked, specifically `./config.yaml`\n",
```

Comment:
This is incorrect, config file paths are different now. Should be updated accordingly

Reply:
Should also explain how and when this environment variable `SAGEMAKER_USER_CONFIG_OVERRIDE` needs to be exported